scripts/klippy-requirements.txt

```diff
@@ -7,3 +7,4 @@ pyserial==3.4
 greenlet==1.1.2
 Jinja2==2.11.3
 python-can==3.3.4
+markupsafe<2.1
```

Comment:
It looks like this got unintentionally got pulled in.  I think Kevin plans to pin markupsafe to 1.1.1.